db/blip_handler.go

```diff
@@ -292,6 +293,9 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 			if !strings.HasPrefix(change.ID, "_") {
 				for _, item := range change.Changes {
 					changeRow := []interface{}{change.Seq, change.ID, item["rev"], change.Deleted}
+					if opts.disableRemovalMessages && change.allRemoved {
```

Comment:
I feel like it would be preferable to do this check at a lower level - indicating whether we want removals in the options we pass to generateBlipSyncChanges, and filter it out at that level.  
@JRascagneres Is this handled by ChangesOptions.Revocations in Lithium for the removals case in ISGR?

Reply:
For revocations it's handled by changes. For removals its done at this level.

Reply:
Ok - it's reasonable to leave where it is, then.